docs/indepth/installation.rst

```diff
@@ -347,10 +347,37 @@ You must also install the ZFS package repository.
 The following commands will install the two repositories and the ``clusterhq-flocker-node`` package.
 Paste them into a root console on the target node:
 
-.. task:: install_flocker
+.. task:: install_flocker fedora-20
    :prompt: [root@node]#
 
-Installing ``clusterhq-flocker-node`` will automatically install Docker, but the ``docker`` service may not have been enabled or started.
+Installing on CentOS 7
+----------------------
+
+Flocker requires the latest available kernel.
```

Comment:
I'm guessing that since CentOS is a fairly stable (ie "slow moving" I suppose) distro, the chances of it introducing a new kernel that's grossly incompatible with ZFS are pretty low and we're counting on that?

What about testing things before our users get them?  I suppose that's what the centos distro-testing build configuration is about - we'll get proposed kernel updates before users of the stable distro do, and if they're broken we have some window to ship a fix...?